modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -188,7 +207,7 @@ struct MatType2
     int _value;
 };
 
-// Universal parameter wrapper for common (pre-defined) and specific (user-defined) parameters
+// Universal parameter wrappers for common (pre-defined) and specific (user-defined) parameters
```

Comment:
Are you sure? I don't think that author means exactly such explanation, let's discuss